Gemfile

```diff
@@ -1,14 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'govuk_app_config'
-gem 'plek'
+gem 'govuk_app_config', '~> 1'
```

Comment:
The syntax for this actually involves having the minor version too. To quote [the docs](http://bundler.io/v1.5/gemfile.html)

> The specifier ~> has a special meaning, best shown by example. ~> 2.0.3 is identical to >= 2.0.3 and < 2.1. ~> 2.1 is identical to >= 2.1 and < 3.0. ~> 2.2.beta will match prerelease versions like 2.2.beta.12.

Reply:
I was reading some different docs that suggested it would work [1].

> It’s also important to know that if you specify a major version only, like this:
> 
> # gemspec
> spec.add_runtime_dependency 'library', '~> 2'
> 
> It will only use the latest version from the 2.x series – so 2.3.0 – and not 3.0.0. This behaviour > may surprise some people, but automatically allowing any major version past version 2 is
> more surprising behaviour.

I realise that bundler and rubygems are different tools, but I was assuming that the semantics of the version constraint syntax was the same?

1: http://guides.rubygems.org/patterns/#pessimistic-version-constraint

Reply:
Oh cool TIL 👍 Apologies

Reply:
No need to apologise. I've just tried it out, specifying '~> 0' for `govuk_app_config to see if it downgrades it, and it does seem to work, which is good.